### PR TITLE
#413 set standard view as default for export command

### PR DESCRIPTION
**Pull request by linhvo**

### CODE REVIEW

File: cmd/export.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ The file does not contain any headers.
 	flags.StringVarP(&Exporter.Host, "host", "", "localhost:10101", "host:port of Pilosa.")
 	flags.StringVarP(&Exporter.Index, "index", "i", "", "Pilosa index to export into.")
 	flags.StringVarP(&Exporter.Frame, "frame", "f", "", "Frame to export into.")
+	flags.StringVarP(&Exporter.View, "view", "v", "", "View to export into - default standard")
```
Comment (travisturner) [inline cmd/export.go]: I think we can remove the word `into` from this line and the lines above.

File: client.go [hunk 2]
```diff
@@ -394,8 +394,8 @@ func (c *Client) ExportCSV(ctx context.Context, index, frame, view string, slice
 		return ErrIndexRequired
 	} else if frame == "" {
 		return ErrFrameRequired
-	} else if view == "" {
-		view = ViewStandard
+	} else if view != ViewInverse || view != ViewStandard {
```
Comment (travisturner) [inline client.go]: won't this condition always return an error?

Comment (linhvo) [inline client.go]: oh right, my bad. I'm fixing it

**Comment by linhvo** [comment]
^ I agree, it's totally better. I'll update as your suggestion then

**Pull request merged**